pom.xml

```diff
@@ -121,16 +121,20 @@ THE SOFTWARE.
     </dependency>
 
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.11</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
       <version>1.9.5</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
```

Comment:
The `objenesis` conflict is with PowerMock. This is pretty standard—every plugin using PowerMock winds up with the same conflict. Ditto `hamcrest-core`, conflicting with a newer one from `junit` / `jenkins-test-harness`.